pkg/systemd/quadlet/quadlet.go

```diff
@@ -1475,14 +1475,7 @@ func ConvertBuild(build *parser.UnitFile, unitsInfoMap map[string]*UnitInfo, isU
 
 	service.AddCmdline(ServiceGroup, "ExecStart", podman.Args)
 
-	service.Setv(ServiceGroup,
-		"Type", "oneshot",
-		"RemainAfterExit", "yes",
-
-		// The default syslog identifier is the exec basename (podman)
-		// which isn't very useful here
-		"SyslogIdentifier", "%N")
-
+	defaultServiceGroup(service, false)
```

Comment:
We should indicate in the release comment that the default behaviour for `.build` files has changed